### DISCUSSION TITLE
Recurrent Survival Analysis

### DISCUSSION THREAD

**Discussion by bachnguyen-tomo** [CamDavidsonPilon/lifelines#1597] [Q&A]
Hello! I wanna know if the library supports recurrent survival analysis, as I saw that an example recurrent SA dataset (holly_molly_polly) is included. I haven't been able to find any code examples online and in the documentation, so if anyone could provide one I would really appreciate it

**Comment by ayushpatnaikgit** [comment]
Same question!
